app/public/js/localSprites.js

```diff
@@ -111,7 +130,16 @@ function updateLocalSprite(localSprite) {
     }
     //rotation. Should there be an "isRotating" flag, ond only do this if it's true?
     localSprite.rotation += localSprite.rotationPerFrame;
+    //WRONG: Both bobbing and scale oscillation should have a nice sinusoid curve to them, but they're just linear right now.
+    //bobbing
+    if (localSprite.bobs) {
```

Comment:
You can get sin motion here by doing:
`localSprite.y = localSprite.originalY + Math.sin(now() / 200) * 10;`

If you change the 200 you can make it bob faster/slower (bigger number will make it bob more slowly), and you can make it oscillate more by changing 10 to a larger number.